app-penetration/sqlmap/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.7.12
+SRCS="tbl::https://github.com/sqlmapproject/sqlmap/archive/$VER.tar.gz"
```

Comment:
Please use Git source instead, as automatically generated tarballs may be re-generated - making them unreliable in the long run.

```
SRCS="git::commit=tags/$VER::https://github.com/sqlmapproject/sqlmap"
CHKSUMS="SKIP"
```